src/pages/waiting-room-to-car/cat-be/components/vehicle-checks/__tests__/vehicle-checks.spec.ts

```diff
@@ -144,6 +139,69 @@ describe('VehicleChecksCatBEComponent', () => {
     });
   });
 
+  describe('invalidVehicleChecks', () => {
+    it('should return vehicle checks as false', () => {
+      const result = component.invalidVehicleChecks(null);
+      expect(result).toEqual({ vehicleChecks: false });
+    });
+  });
+
+  describe('validateVehicleChecks', () => {
+    it('should call invalidVehicleChecks() if all questions have not been answered', () => {
+      const formBuilder: FormBuilder = new FormBuilder();
+      component.formGroup = formBuilder.group({
+        vehicleChecksSelectQuestions: null,
+      });
+      spyOn(component, 'everyQuestionHasOutcome').and.returnValue(false);
+      spyOn(component, 'invalidVehicleChecks');
+      component.validateVehicleChecks();
+      expect(component.invalidVehicleChecks).toHaveBeenCalled();
+    });
+  });
```

Comment:
a bit pedantic, but feel like incompleteVehicleChecks is a much better name for this.

Reply:
Yep, I agree.